tasks/main.yml

```diff
@@ -0,0 +1,108 @@
+---
+- name: Create a unix group
+  group:
+    name: jira
+    system: true
+
+- name: Create a unix user
+  user:
+    name: jira
+    group: jira
+    comment: JIRA system user
+    shell: /bin/false
+    home: /opt/atlassian-jira
+    createhome: no
+    system: true
+
+- name: Create installation directory
+  file:
+    path: /opt/atlassian-jira
+    state: directory
+    owner: root
+    group: jira
+    mode: 0755
+
+- name: Create data directories
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: root
+    group: jira
+    mode: 0770
+  with_items:
+    - /var/lib/jira
+    - /var/log/jira
+
+- name: Download archive
+  get_url:
+    url: https://www.atlassian.com/software/jira/downloads/binary/atlassian-jira-core-{{ jira_version }}.tar.gz
```

Comment:
Please download the archive to the local machine and delete it afterwards.
The machine hosting JIRA may not have an internet connection.

Reply:
It will have an internet connection to contact the addons server. We are required to have this to install the functionality we actually use